### PR TITLE
Score holder

### PR DESCRIPTION
**Pull request by s373r**

### CODE REVIEW

File: src/renderer.py [hunk 1]
```diff
@@ -28,6 +31,9 @@ def __draw_cell(self, x: int, y: int, is_fill: bool) -> None:
                             y + self.__BORDER_WEIGHT,
                             self.__CELL_CHARACTER[is_fill])
 
+    def __draw_string(self, x: int, y: int, value) -> None:
```
Comment (alena-bartosh) [inline src/renderer.py]: `value` is a some string, correct?

Comment (alena-bartosh) [inline src/renderer.py]:
```suggestion
    def __draw_string(self, x: int, y: int, value: str) -> None:
```

File: src/score_holder.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def increment(self, burned_line_count: int) -> None:
+        if burned_line_count not in self.__LINE_SCORE_INCREMENT_MAPPING:
+            raise ValueError(f'Invalid burned_line_count [{burned_line_count}]!')
+
+        score_increment = self.__LINE_SCORE_INCREMENT_MAPPING[burned_line_count]
+
+        self.__scores += score_increment
+        self.__burned_line_count += burned_line_count
+
+    def get_score(self) -> int:
```
Comment (alena-bartosh) [inline src/score_holder.py]:
```suggestion
    def get_scores(self) -> int:
```

**Comment by alena-bartosh** [comment]
Please add screenshot with new scores string :)

File: src/renderer.py (hunk 1 above)
Comment (s373r) [reply src/renderer.py]: yes, I'll add type

Comment (s373r) [reply src/renderer.py]: Done

Comment (s373r) [reply src/renderer.py]: done

**Review by alena-bartosh — APPROVED** [review]
Very good!

**Comment by Serg-Bartosh** [comment]
this is no longer relevant, you can put it all in the folder 'utility' 

:clown_face:

**Review by Serg-Bartosh — APPROVED** [review]
keep writing like this - it's cool :cupid:

**Pull request merged**